File: entrega2/menu/src/main/java/com/aps/menu/entidades/Conta.java
package com.aps.menu.entidades;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
public class Conta {    
    @Id
    protected Long id;
    protected String email;
    protected String senha;


    public Conta(Long id, String email, String senha) {
        this.setId(id);
        this.setEmail(email);
        this.setSenha(senha);
    }

    public Conta () {
        
    }

    public Long getId() {
        return id;
    }


    public void setId(Long id) {
        this.id = id;
    }


    public String getEmail() {
        return email;
    }


    public void setEmail(String email) {
        this.email = email;
    }

    public String getSenha() {
        return senha;
    }

    public void setSenha(String senha) {
        this.senha = senha;
    }    

}

File: rup/menu-app/src/entidades/conta.ts
export class Conta {
  protected id: number;
  protected email: string;
  protected senha: string;
  
  constructor(id: number, email: string, senha: string) {
    this.id = id;
    this.email = email;
    this.senha = senha;
  }

  public setId(id: number) {
    this.id = id;
  }

  public getId() : number {
    return this.id;
  }

  public getEmail() : string {
    return this.email;
  }

  public getSenha() : string {
    return this.senha;
  }
}